### PR TITLE
New translation for import, settings and adressbook name & new import button design

### PR DESCRIPTION
**Pull request by skjnldsv**
Added some translation to the import button, the "add addressbook" input and the settings button.

@DeepDiver1975

### CODE REVIEW

File: js/components/addressBookList/addressBookList_controller.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ angular.module('contactsApp')
 		ctrl.loading = false;
 	});
 
+	ctrl.t = {
+		AdressbookName : t('contacts', 'Adressbook name')
```
Comment (DeepDiver1975) [inline js/components/addressBookList/addressBookList_controller.js]:
``` js
addressBookName : t('contacts', 'Address book name')
```

File: l10n/en_GB.js [hunk 2]
```diff
@@ -36,6 +36,9 @@ OC.L10N.register(
     "Fax home" : "Fax home",
     "Fax work" : "Fax work",
     "Pager" : "Pager",
-    "Voice" : "Voice"
+    "Voice" : "Voice",
```
Comment (DeepDiver1975) [inline l10n/en_GB.js]: please remove all changes to tranlsation file - THX

File: templates/addressBookList.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 	<li ng-repeat="addressBook in ctrl.addressBooks" addressbook data="addressBook"></li>
 	<li>
 		<form ng-submit="ctrl.createAddressBook()">
-			<input id="newList" placeholder="Adressbook name" class="newAddressBookInput" ng-model="ctrl.newAddressBookName" type="text" />
+			<input id="newList" placeholder="{{ctrl.t.AdressbookName}}" class="newAddressBookInput" ng-model="ctrl.newAddressBookName" type="text" />
```
Comment (DeepDiver1975) [inline templates/addressBookList.html]: ctrl.t.addressBookName

**Comment by skjnldsv** [comment]
Do you want me to add #262 ?

**Comment by DeepDiver1975** [comment]
> Do you want me to add #262 ?

This is already in here

**Comment by skjnldsv** [comment]
Css is missing.

**Comment by DeepDiver1975** [comment]
> Css is missing.

no

**Comment by skjnldsv** [comment]
> no

I need to buy new eyes... Okay, will edit the commits when back from work.
Thanks DeepDiver.

EDIT: all done and rebased.

**Comment by DeepDiver1975** [comment]
Looks better now .... let me test ...

**Comment by DeepDiver1975** [comment]
there are conflicts in here - let's merge this one: https://github.com/owncloud/contacts/pull/271

**Pull request closed without merging**